README.md

```diff
@@ -1 +1,121 @@
-# python-SSL-checker
\ No newline at end of file
+## Python SSL checker
+
+---
+This project validate SSL certificate external site. If the certificate has expired then it is sent notification message on you Telegram bot otherwise the script shows the number of days before the expiration date.
+
+---
+
+### Requirements
```

Comment:
Requirements should be also listed in a separated files requirements.txt
Check pip freeze command